system-addon/lib/ActivityStream.jsm

```diff
@@ -56,10 +56,10 @@ const PREFS_CONFIG = new Map([
       provider_icon: "pocket",
       provider_name: "Pocket",
       read_more_endpoint: "https://getpocket.com/explore/trending?src=ff_new_tab",
-      stories_endpoint: `https://getpocket.com/v3/firefox/global-recs?consumer_key=$apiKey&locale_lang=${args.locale}`,
+      stories_endpoint: `https://getpocket.com/v3/firefox/global-recs?version=2&consumer_key=$apiKey&locale_lang=${args.locale}`,
```

Comment:
$apiKey is fine. This is replaced with the API key in TopStoriesFeed, not when the string is created/interpolated i.e. apiKey is not a variable in scope here, but is read from a pref.